src/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSource.cs

```diff
@@ -18,6 +19,7 @@ public partial class DiagnosticListener : System.Diagnostics.DiagnosticSource, S
   }
   public abstract partial class DiagnosticSource {
     protected DiagnosticSource() { }
+    public static System.Diagnostics.DiagnosticSource DefaultSource { get { return default(System.Diagnostics.DiagnosticSource); } }
```

Comment:
Didn't we just remove both of these in https://github.com/dotnet/corefx/commit/8a39134f5b6058ba4ec1b01153ee78597b9ba295?  Are they being added back on purpose?